cmd/juju/application/deploy.go

```diff
@@ -369,6 +369,46 @@ Constraints can be specified by specifying the '--constraints' option. If the
 application is later scaled out with ` + "`juju add-unit`" + `, provisioned machines
 will use the same constraints (unless changed by ` + "`juju set-constraints`" + `).
 
+Application configuration values can be specified using '--config' option. This
+option accepts either a path to a yaml-formatted file or a key=value pair. 
+Configurtion file provided should be in format
+<charm name>:
+	<option name>: <option value>
+	...
+For example, to deploying 'mediawiki' with the configuration file 'mycfg.yaml'
+that contains:
+
+mediawiki:
+	name: my media wiki
+	admins: me:pwdOne
+	debug: true
```

Comment:
Given this is enshrouded in history, I wonder if we should use "postgresql" as its a much better written charm.